src/components/interaction-designer/Block.vue

```diff
@@ -64,6 +64,7 @@
                              :id="`exit/${exit.uuid}/pseudo-block-handle`"
                              :key="`exit/${exit.uuid}/pseudo-block-handle`"
                              :is-editable="isEditable"
+                             v-b-tooltip.hover.bottom="transIf(isEditable, 'flow-builder.tooltip-new-connection')"
```

Comment:
Can we bump these exit tooltips to be up on top rather than bottom? 😻 or does it look bad? It feels a bit odd not being able to see the connection coming out the bottom. 

If it came out the top, would we want it to appear above the exit label or the button?

Reply:
Ok, let me just put it on `top` for now